docs: Remove two pane layout from quickstarts and rearrange the content (#47398)
supabase/supabase: apps/docs/content/guides/getting-started/quickstarts/ruby-on-rails.mdx

title: 'Use Supabase with Ruby on Rails'
subtitle: 'Learn how to create a Rails project and connect it to your Supabase Postgres database.'
breadcrumb: 'Framework Quickstarts'
---

## 1. Create a Rails project

> Make sure your Ruby and Rails versions are up to date, then use `rails new` to scaffold a new Rails project. Use the `-d=postgresql` flag to set it up for Postgres.

Reviewer: ```suggestion
With your Ruby and Rails versions up to date, run `rails new` on your terminal to scaffold a new project.

Use the `-d=postgresql` flag to set it up for Postgres.
```

> Go to the [Rails docs](https://guides.rubyonrails.org/getting_started.html) for more details.

Reviewer: ```suggestion
Check the [Rails docs](https://guides.rubyonrails.org/getting_started.html) for more details.
```
